django/db/models/base.py

```diff
@@ -1423,6 +1424,12 @@ def _check_model_name_db_lookup_clashes(cls):
             )
         return errors
 
+    @classmethod
```

Comment:
And I'd put the actual method after `_check_unique_together` too, to match the order above :)

Reply:
Agree, thanks!